doc/lightning-utxopsbt.7.md

```diff
@@ -0,0 +1,60 @@
+lightning-utxopsbt -- Command to populate PSBT inputs from given UTXOs
+================================================================
+
+SYNOPSIS
+--------
+
+**utxopsbt** *satoshi* *feerate* *startweight* *utxos* \[*reserve*\]
+
+DESCRIPTION
+-----------
+
+*utxopsbt* is a low-level RPC command which creates a PSBT using unreserved
+inputs in the wallet, optionally reserving them as well.
+
+It deliberately mirrors the parameters and output of
+lightning-fundpsbt(7) except instead of an optional *minconf*
+parameter to select unreserved outputs from the wallet, it takes a
+compulsory list of outputs to use.
+
+*utxos* must be an array of "txid:vout", each of which must be
+reserved or available: the total amount must be sufficient to pay for
+the resulting transaction plus *startweight* at the given *feerate*,
+with at least *satoshi* left over (unless *satoshi* is **all**, which
+is equivalent to setting it to zero).
+
+RETURN VALUE
+------------
+
+On success, returns the *psbt* containing the inputs, *feerate_per_kw*
+showing the exact numeric feerate it used, *estimated_final_weight* for
+the estimated weight of the transaction once fully signed, and
+*excess_msat* containing the amount above *satoshi* which is
+available.  This could be zero, or dust.  If *satoshi* was "all",
+then *excess_msat* is the entire amount once fees are subtracted
+for the weights of the inputs and *startweight*.
```

Comment:
On Elements, do either `fundpsbt` or `utxopsbt` add a fee output?  Or does the caller have to do that?

Reply:
If caller has to do that, I suggest adding this explicitly to the documentation, and recommending that if the output is below a dust limit (which should be documented as well) then it should be added to the fee output as well.

Reply:
I don't know?  @cdecker and @niftynei have been tracking this...

Reply:
`bitcoin_tx_finalize` does this automatically, but third parties outside of `lightningd` should probably know.

Reply:
@niftynei finally checked in on this and discovered that no, `fundpsbt` does not add a fee output for the elements case and it definitely should. made an issue #3940 to track this (it currently breaks one of our CI tests because of this)